application/templatetags/app_filters.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+from datetime import date, timedelta
+
+register = template.Library()
+
+@register.filter
+def obfuscate_email(value): # Only one argument.
```

Comment:
maybe rename this to "hide"? "Obfuscate" means to make confusing, but the data is still there.